Reflection_Engine/Query/BHoMMethodList.cs

```diff
@@ -45,33 +45,6 @@ public static List<MethodInfo> BHoMMethodList()
             return m_BHoMMethodList;
         }
 
-        /***************************************************/
-
-        public static List<MethodBase> AllMethodList()
```

Comment:
Same thing here. clarity of the code is reduced by moving methods in separate files. The file name was allowing for those methods to be together.